matching-service/docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: '3.9'
+
+services:
+  matching-service:
+    image: matching-service:latest
+    build: .
+    ports: 
+      - 3000:3000
```

Comment:
The frontend is already using port 3000. Perhaps 8001 would be a good alternative